src/Microsoft.Framework.PackageManager/Publish/PublishProject.cs

```diff
@@ -118,8 +118,12 @@ private bool EmitNupkg(PublishRoot root)
             buildOptions.ProjectDir = project.ProjectDirectory;
             buildOptions.OutputDir = Path.Combine(project.ProjectDirectory, "bin");
             buildOptions.Configurations.Add(root.Configuration);
-            buildOptions.Reports = root.Reports;
             buildOptions.GeneratePackages = true;
+            buildOptions.Reports = root.Reports.ShallowCopy();
+
+            // Mute "dnu pack" completely if it is invoked by "dnu publish --quiet"
+            buildOptions.Reports.Information = root.Reports.Quiet;
```

Comment:
Without this, you'll see
![publishmessage_extrabuild](https://cloud.githubusercontent.com/assets/1383883/7760859/6860fc66-ffd4-11e4-9be4-efa6de9de865.PNG)